src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -130,14 +130,16 @@ public void CompileMethod(MethodCodeNode methodCodeNodeNeedingCode)
 
                     if (!_compilation.Options.NoLineNumbers)
                     {
-                        IEnumerable<ILSequencePoint> ilSequencePoints = typeSystemContext.GetSequencePointsForMethod(MethodBeingCompiled);
+                        IEnumerable<ILSequencePoint> ilSequencePoints =
+                            typeSystemContext.GetSequencePointsForMethod(MethodBeingCompiled, _compilation.GetMethodIL(MethodBeingCompiled));
```

Comment:
Change the return type of `Get_CORINFO_METHOD_INFO` to `MethodIL` so that we do not have to re-fetch it here?